src/main/java/hudson/security/AuthorizationMatrixProperty.java

```diff
@@ -300,9 +306,9 @@ public Object unmarshal(HierarchicalStreamReader reader,
 				reader.moveDown();
 				reader.getValue(); // we used to use this but not any more.
 				reader.moveUp();
-				prop = reader.peekNextChild();
+				prop = reader.peekNextChild(); // We check the next field
 			}
-			else if ("blocksInheritance".equals(prop)) {
```

Comment:
Seems like a non-findbugs related change... not saying it is incorrect... rather that it seems like one